indico/modules/events/contributions/blueprint.py

```diff
@@ -103,6 +103,10 @@
 _bp.add_url_rule('/manage/contributions/types/<int:contrib_type_id>/delete', 'delete_type',
                  management.RHDeleteContributionType, methods=('POST',))
 
+# Default contribution duration
+_bp.add_url_rule('/manage/contributions/duration/', 'manage_default_duration',
```

Comment:
`/manage/contributions/default-duration` (no trailing slash if the url segment has nothing inside)